css/main.css

```
@@ -8,3 +8,43 @@ body {
   color: #3b3d40;
   font-size: 14px;
 }
+
+footer {
```

Não usar tag no css, é sempre bom usar class, por boas práticas 
```suggestion
.footer {
```